### PR TITLE
 Added duplet generation

### PR DESCRIPTION
**Pull request by ffinfo**
Depends on https://github.com/biopet/10xkit/pull/3

### CODE REVIEW

File: src/test/scala/nl/biopet/tools/tenxkit/mergebams/MergeBamsTest.scala [hunk 1]
```diff
@@ -32,4 +37,204 @@ class MergeBamsTest extends ToolTest[Args] {
...
+      ))
+
+    val reader = SamReaderFactory
+      .makeDefault()
+      .validationStringency(ValidationStringency.SILENT)
+      .open(outputBam)
+    val it1 = reader.iterator()
+    val aim = 40000 * fraction
+    val reads = it1.size
+    // the 0.02 means that the result of the downsampling is variable with 2%
```
Comment (rhpvorderman) [inline src/test/scala/nl/biopet/tools/tenxkit/mergebams/MergeBamsTest.scala]: You may be either too lenient or to strict here with the 2% here depending on the fraction chosen. 
This means tests will randomly fail during CI.

It is best to set up the boundaries using a binomial distribution and test whether the aim is within those boundaries.

Comment (rhpvorderman) [inline src/test/scala/nl/biopet/tools/tenxkit/mergebams/MergeBamsTest.scala]: You may be either too lenient or to strict here with the 2% here depending on the fraction chosen.
This means tests will randomly fail during CI.

It is best to set up the boundaries using a binomial distribution and test whether the aim is within those boundaries.

**Review by rhpvorderman — CHANGES_REQUESTED** [review]

File: src/test/scala/nl/biopet/tools/tenxkit/mergebams/MergeBamsTest.scala (hunk 1 above)
Comment (ffinfo) [reply src/test/scala/nl/biopet/tools/tenxkit/mergebams/MergeBamsTest.scala]: The random is seeded so it should give the same result always.

Comment (rhpvorderman) [reply src/test/scala/nl/biopet/tools/tenxkit/mergebams/MergeBamsTest.scala]: Okay!

**Review by rhpvorderman — APPROVED** [review]

**Pull request merged**